src/domain/entities/TranslatableText.ts

```diff
@@ -16,3 +17,24 @@ export const buildTranslate = (locale: string): TranslateMethod => {
 };
 
 export type TranslateMethod = (string: TranslatableText) => string;
+
+export const importTranslate = <T extends TranslatableText>(item: T, language: string, term: string | undefined): T => {
```

Comment:
When we have several arguments we tend to wrap them in an object to have more declarative arguments when the function is called like `importTranslations({ language: model.contents (for example), ... })`